### ISSUE TITLE
Can't import AutoMLRun - getting AttributeError 'tensorflow...'

### ISSUE THREAD

**Issue by arielcedola** [Azure/MachineLearningNotebooks#344]
Hi,

since Saturday April 27th (yes, the Bootcamp day) I started to have problems to run autoML experiments on Azure Notebooks, the same experiments that until the day before were working ok. Any time I try to run the first cell, the statement:

from azureml.train.automl.run import AutoMLRun

produces the error message:

AttributeError: module 'tensorflow.python.ops.nn' has no attribute 'selu'

Any idea about what's happening? Is there some library that should be updated? Thanks

**Comment by skasturi** [comment]
@arielcedola, Can you please post the version of the SDk you are using?

**Comment by gaugup** [comment]
Which version of tensorflow do you see on AZN?

**Comment by arielcedola** [comment]
Thank you for your replies, surprisingly Azure Notebooks is working ok now, at least for me.
I want to share with you my concern about this situation. The AutoML feature stopped working precisely the morning of the Global Azure Bootcamp, the day in which I had to give my session about Azure ML and in particular AutoML. Obviously this not prevented the normal development of the presentation, I ran the demos locally, but without doubts it was hampered by this situation. Something was wrong the wrong day.
SDK version was 1.0.17 and tf version 1.12.0.
